docs/RELEASE_NOTES_DRAFT

```diff
@@ -5,5 +5,10 @@ Release vX.XX.XX
 Bug Fixes:
 
 Features:
+ - Add a bundle-remove --recursive flag to remove a bundle and all its dependencies from
```

Comment:
Can you try to summarize the feature in just short topic? The idea is to say what the release introduce, not necessary how.

Reply:
🤔 I'll try to reword this in a shorter way

Reply:
done, it will be in the next update